### ISSUE TITLE
Update external links translations

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#9333]
**Is your feature request related to a problem? Please describe.**

> As a WS screenreader user
> I want external links to be identified as <external> in my language
> so that I know if I click on the link I'll be taken from the BBC website

Currently some of the translations for WS are "external link" where the translation should be "external" as the screen reader adds the semantics of 'link' itself.

Some users are having the experience where the word link is announced twice, once from the translation and once from the screen reader semantics e.g. 'Google, external link, link'.

Depending on the screen reader it should be announced as 'Google, external, link' or 'Link, Google, external')

**Describe the solution you'd like**
Update the services configs for external links with the label "External" in the spreadsheet here on line 32 where required: https://docs.google.com/spreadsheets/d/1f1x53UDjmzJCf0Pv94t1AOPceCapolFxeu77LMyA4IM/edit?usp=sharing

the RtoL languages will need extra checks with editorial for the ordering of the linktext 

<google-sheets-html-origin><style type="text/css"></style>

Service|Text|RtoL
---|---|---
Persian | [linktext]، خارجی |RtoL
Igbo | [linktext], na mpụta
Pidgin | [linktext], outside
Yoruba | [linktext], ìta
Urdu | [linktext], بیرو |RtoL
Pashto | بهرنی [linktext] |RtoL 
Arabic | [linktext]، خارجي |RtoL
Thai | [linktext] จากภายนอก
Punjabi | [linktext], ਬਾਹਰੀ
Afaan Oromoo | [linktext], alaan
Amharic | [linktext], ውጫዊ
Indonesian | [linktext], eksternal
Tigrinya | [linktext] ደጋዊ
Korean | [linktext], 외부 사이트
Afrique | [linktext], externe
Gahuza | [linktext], bivuye ahandi
Somali | [linktext], kale
Swahili | [linktext], ya nje
Azeri | [linktext], BBC-dən kənar
Hausa | [linktext], waje
Vietnamese | [linktext], bên ngoài
Kyrgyz | [linktext], баракчалар
Japanese | [linktext]は外部ページです
Nepali | [linktext], बाह्य
Uzbek (Cyrillic) | [linktext], ташқи
Uzbek (Latin) | [linktext], tashqi
Sinhala | [linktext], බාහිර
Bengali | [linktext], বাইরের
UK China (Simp) | [linktext], 外部
UK China (Trad) | [linktext], 外部
Turkce | [linktext], dış
Burmese (Unicode) | [linktext] ပြင်ပစာမျက်နှာ
Cymru Fyw | [linktext], dolen allanol
Gujarati | [linktext] બહારની
Marathi | [linktext], बाहेर
BBC News Mundo | [linktext], externo
Naidheachdan | [linktext], taobh a-muigh
Portuguese | [linktext], externo
Russian | [linktext], внешняя
Serbian(Cyr) | [linktext], спољна
Serbian (Lat) | [linktext], spoljna
Tamil | [linktext], வெளி
Telugu | [linktext], ఇతర
Ukrainian | [linktext], зовнішнє
Russian for the Ukrainian service | [linktext], внешняя  
Hindi | [linktext], बाहरी
Zhongwen (Simp) | [linktext], 外部
Zhongwen (Trad) | [linktext], 外部

**Describe alternatives you've considered**

**Expected behaviour:**
Given I use a screenreader 
When I navigate to an external link on an optimo page
it should announce the link name, external and the screen reader semantics of link
and external should be in the language of that article

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by ryanmccombe** [comment]
These translations are in the `externalLinkText` of the service configs: https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/afrique.js#L28

**Comment by JonBeeb** [comment]
Confirmed with editorial that it is ok to use the Russian value for the 'Russian for the Ukrainian' in the Ukraine service.

**Comment by aeroplanejane** [comment]
Looks like the translation updates have resolved this for articles e.g. https://www.test.bbc.com/mundo/articles/c9y2p60m620o
